src/main/java/Duke.java

```diff
@@ -1,10 +1,115 @@
+import java.util.Scanner;
+
 public class Duke {
+    // Constants
+    private static final String MESSAGE_BOUNDARY = "____________________________________________________________";
+
+    // Attributes
+    private static Task[] tasks = new Task[100];
+    private static int numoftask = 0;
+
+    // Constructor
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        Greet();
+        Echo();
     }
+
+    public static void Greet(){
+        String lines = "____________________________________________________________";
+        String greet = " Hello! I'm Duke\n"
+                    + " What can I do for you?\n";
+        System.out.println(MESSAGE_BOUNDARY + "\n" + greet + MESSAGE_BOUNDARY);
+    }
+
+    public static void Echo(){
+        boolean continueChat;
+        Scanner in = new Scanner(System.in);
+
+        do{
+            // Get query from user
+            String userInput = in.nextLine();
+
+            // Give response
+            continueChat = giveResponse(userInput);
+
+        }while(continueChat);
+
+        in.close();
+    }
+
+    public static boolean giveResponse(String userInput){
```

Comment:
Maybe add a Javadoc comment to explain what this method does? This would help readers quickly understand the code.